tools/st2_deploy.sh

```diff
@@ -97,6 +99,22 @@ else
     MISTRAL_STABLE_BRANCH="st2-0.5.1"
 fi
 
+update_hosts_file()
+{
+  if [ -z "`cat /etc/hosts | grep -e '127.0.0.1\slocalhost'`" ]; then
```

Comment:
> Are we sure about this change? I don't think we should be changing anyones /etc/hosts without notifying them - if I have some terrible reason for this configuration I would hate it if some deploy script went in an changed things!

I'm open to suggestions. I don't know how else to fix this on vagrant w/ chef/fedora-20.